lib/dai-plugin-mcd/src/ManagedCdp.js

```diff
@@ -12,17 +13,24 @@ export default class ManagedCdp {
 
     assert(ilk && typeof ilk === 'string', 'Must specify ilk');
     this.ilk = ilk;
-
     this._cdpManager = cdpManager;
-    this.currency = cdpManager
-      .get(ServiceRoles.CDP_TYPE)
-      .getCdpType(null, ilk).currency;
+    this._cdpType = cdpManager.get(ServiceRoles.CDP_TYPE).getCdpType(null, ilk);
+    this.currency = this._cdpType.currency;
   }
 
-  async getCollateralValue() {
+  async getCollateralAmount() {
     return this.currency.wei((await this._urnInfo()).ink);
   }
 
+  async getCollateralValue() {
+    const [collateral, price] = await Promise.all([
+      this.getCollateralAmount(),
+      this._cdpType.getPrice()
+    ]);
+    const ratio = createCurrencyRatio(USD, this.currency);
+    return ratio(collateral.times(price));
```

Comment:
this should just be `collateral.times(price)`, no wrapper, so it ends up as USD (since it's DAI * USD/DAI)